app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -68,6 +69,8 @@ class HomeFragment : Fragment() {
 
         setupHomeMenu()
 
+        sessionsComponent.view.visibility = if ((activity as HomeActivity).browsingModeManager.isPrivate)
+            View.INVISIBLE else View.VISIBLE
```

Comment:
We'll want to use `View.GONE` here. [Here is a good example that explains the difference](https://stackoverflow.com/a/23211042) :+1:

Reply:
Good catch!